scripts/prepare-e2e.cts

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import fse from 'fs-extra';
+import * as execa from 'execa';
+import prettier from 'prettier';
+
+const prepareCommands = [
```

Comment:
I recommend to create a playground and install depenpencies  together with island.js so it will be faster in local test.

Reply:
> I recommend to create a playground and install depenpencies together with island.js so it will be faster in local test.

Let me see.